.shellcheckrc

```diff
@@ -0,0 +1,32 @@
+<!-- until https://github.com/checkstyle/checkstyle/issues/11637 -->
+# ~/.shellcheckrc
```

Comment:
remove `~/.shellcheckrc`

make `# until ....`

`<!-- ` is comment only in xml

Reply:
Changes Done

Reply:
@romani It seems that only my PR is getting flagged. 
https://app.travis-ci.com/github/checkstyle/checkstyle/requests

Reply:
https://app.travis-ci.com/github/checkstyle/checkstyle/requests

> Abuse detected

Reply:
I need to write to support

Reply:
Oh. Kindly let us know what was the issue which made this happen when they get back to you.

Reply:
@romani  IF we will use Terminal then we can only check single file errors.
For eg: shellcheck common.sh We can only see the errors of common.sh

As far as I know, We can't see all the errors of a particular shellcheck code using the terminal.

Reply:
It is ok for now, we just need alert that something is wrong. So contributors can do multiple iterations to fix violations or run it on local and resolve all on local.

Please rebase your PR, Travis team cleared your

Reply:
PR REBASED.

@romani I am just curious to know what was the reason behind my PR getting flagged?

Reply:
I didn't disclose, it is security alert, they will not give details.